src/pages/index.hbs

```diff
@@ -24,6 +29,7 @@
 
     .splashPage > .features > li {
       margin: 0 20px;
+      min-width: 11rem;
```

Comment:
I'm going to change this to a `px` value, as `px` is already adaptive to different pixel densities and stuff; it's not actually 1 pixel.